lib/patch-series.ts

```diff
@@ -286,7 +287,13 @@ export class PatchSeries {
                             basedOn = match2[2];
                             break;
                         case "cc:":
-                            cc.push(match2[2]);
+                            addressparser(match2[2]).forEach((e) => {
+                                if (e.name) {
+                                    cc.push(e.name + " <" + e.address + ">");
```

Comment:
Maybe use a template here instead of `+`? 
```
`${e.name} <${e.address}>`
```
Or remove the if with a conditional
```
cc.push(`${e.name ? `${e.name} <${e.address}>` : `${e.address}`}`);
```

Reply:
Good idea about the simplification. Personally, I like ternaries, but in this instance it seems a bit harder to read, so I refrained from doing it.

Reply:
My example didn't help when it should have been `: e.address` but no problem.  It is already well indented.